tests/README.rst

```diff
@@ -6,18 +6,20 @@ Crate Docker Image Development
 Running Tests
 -------------
 
-This project uses `buildout <https://pypi.python.org/pypi/zc.buildout/>`_
-to setup the testing environment for ``docker-crate``.
+Create a virtual environment::
 
-Run ``bootstrap.py``::
+  >>> python3 -m venv docker-crate_env
 
-  >>> python3.4 bootstrap.py
+Then activate it::
 
-And afterwards run buildout::
+  >>> chmod 755 docker-crate_env/bin/activate
+  >>> . docker-crate_env/bin/activate
```

Comment:
More common is to source it:

```
>>> source docker-crate_env/bin/activate
```

Then the `chmod 755` step is not required.
We usually also call it just `venv`, `.venv` or `env`